packages/scenes/src/components/VizPanel/VizPanel.tsx

```diff
@@ -245,6 +254,33 @@ export class VizPanel<TOptions = {}, TFieldConfig extends {} = {}> extends Scene
     return sceneTimeRange.state.value;
   };
 
+  public async changePluginType(pluginId: string, options?: DeepPartial<{}>, newFieldConfig?: FieldConfigSource) {
```

Comment:
```suggestion
  public async changePluginType(pluginId: string, newOptions?: DeepPartial<{}>, newFieldConfig?: FieldConfigSource) {
```
I understand, when changing the plugin type, we want to apply new options and field config to the new plugin, right? I'd try to be kind of specific for people using the changePluginType in their apps :D